pkg/intermediate/aggregate_test.go

```diff
@@ -265,16 +265,34 @@ func TestCorrelateRecords(t *testing.T) {
 	}
 }
 
-func TestDeleteTupleFromMap(t *testing.T) {
+func TestDeleteTupleFromMapWithLock(t *testing.T) {
```

Comment:
Not in this PR, can you help change Tuple -> FlowKey in test names for this and next testing function? Thanks!